sgp4/model.py

```diff
@@ -89,6 +89,90 @@ def sgp4_array(self, jd, fr):
         r.shape = v.shape = len(jd), 3
         return e, r, v
 
+    def export_tle(self):
```

Comment:
Since this method does not access any private attributes of the TLE, or modify its state, I think we can simplify here by making it a plain function. Could you create a new module `export.py` and place this there as a plain function `export_tle(satrec)` that takes the satrec as its argument? Thanks!

Reply:
Done! I will commit this file along with the str.format() changes you suggested.